js/packages/phoenix-mcp/src/supportTools.ts

```diff
@@ -0,0 +1,287 @@
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import z from "zod";
+import { RUNLLM_CONFIG, MCP_CONSTANTS } from "./constants.js";
+
+const PHOENIX_SUPPORT_DESCRIPTION = `Get help and support for Arize Phoenix and AI observability questions.
+
+This tool connects to Arize's specialized AI assistant that can answer questions about:
+- Arize Phoenix observability platform
+- LLM monitoring and evaluation
+- AI model performance analysis
+- Tracing and debugging AI applications
+- Phoenix datasets, experiments, and prompt management
+- Best practices for AI observability
+
+Use this tool when you need expert assistance with Arize Phoenix features, troubleshooting,
+or general AI observability guidance.
+
+Example usage:
+  - "How do I set up tracing for my LLM application?"
+  - "What are the best practices for evaluating RAG systems?"
+  - "How do I use Phoenix datasets for prompt engineering?"
+  - "What metrics should I track for my AI model?"
+
+Expected return:
+  Expert guidance and information about Arize Phoenix and AI observability practices.`;
+
+interface MCPRequest {
+  jsonrpc: string;
+  id: number;
+  method: string;
+  params?: Record<string, unknown>;
+}
+
+interface MCPResponse {
+  jsonrpc: string;
+  id: number;
+  result?: {
+    tools?: Tool[];
+    content?: ContentItem[];
+    [key: string]: unknown;
+  };
+  error?: {
+    code: number;
+    message: string;
+  };
+}
+
+interface Tool {
+  name: string;
+  description?: string;
+  inputSchema?: Record<string, unknown>;
+}
+
+interface ContentItem {
+  type: string;
+  text?: string;
+}
+
+async function runLLMEndpoint(question: string): Promise<string> {
+  const url = RUNLLM_CONFIG.ENDPOINT;
+  const headers = RUNLLM_CONFIG.HEADERS;
+
+  try {
+    // Skip tool discovery for performance - we know 'chat' exists
+    // ------------------------------------------------------------------
+    // Call the 'chat' tool directly with the provided message
+    // ------------------------------------------------------------------
+    const callRequest: MCPRequest = {
+      jsonrpc: MCP_CONSTANTS.JSONRPC_VERSION,
+      id: 1,
+      method: MCP_CONSTANTS.METHODS.TOOLS_CALL,
+      params: {
+        name: "chat",
+        arguments: { message: question },
+      },
+    };
+
+    const chatResponse = await fetch(url, {
+      method: "POST",
+      headers,
+      body: JSON.stringify(callRequest),
+      signal: AbortSignal.timeout(60000), // 60 second timeout
+    });
+
+    if (!chatResponse.ok) {
+      throw new Error(`HTTP error! status: ${chatResponse.status}`);
+    }
+
+    const chatContentType = chatResponse.headers.get("content-type");
+
+    if (chatContentType?.includes("text/event-stream")) {
+      // Handle SSE stream for chat response with timeout
+      if (!chatResponse.body) {
+        throw new Error("No response body");
```

Comment:
This error is super generic - you might want to make it hyper specific for debugging